src/cluster.tsx

```diff
@@ -150,12 +154,46 @@ export default class Cluster extends React.Component<Props, State> {
       .map((leave: Feature) => this.featureClusterMap.get(leave));
   };
 
+  public zoomToClusterBounds = (event: React.MouseEvent<HTMLElement>) => {
```

Comment:
Destruct event? `{ currentTarget, target }`

Reply:
Yeah, but it's not very convenient to type `{ currentTarget, target }` instead of `event`

Reply:
It is not a big deal, happy with both to be honest